securedrop/sass/_base.sass

```diff
@@ -348,9 +366,7 @@
 
   .logo
     width: 250px
-    margin-right: 20px
-    margin-top: 36px
-    margin-left: 9px
+    margin-right: 20px 36px 20px auto
```

Comment:
You meant **margin** here or is it really **margin-right** ?

Reply:
Yep. I did mean that.